h2o-hadoop-2/h2o-mapreduce-2-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -630,6 +632,7 @@ static void usage() {
                     "          [-jobname <name of job in jobtracker (defaults to: 'H2O_nnnnn')>]\n" +
                     "              (Note nnnnn is chosen randomly to produce a unique name)\n" +
                     "          [-principal <kerberos principal> -keytab <keytab path> [-run_as_user <impersonated hadoop username>] | -run_as_user <hadoop username>]\n" +
+                    "          [-hiveHost <hostname:port> -hivePrincipal <hive server principal>]\n" +
```

Comment:
Not sure if we have to put it here. Not all options are documented. If we do include it in the help we should mark it as experimental (it could change).

Reply:
removed